bindings/python/Android.mk

```diff
@@ -74,6 +77,26 @@ LOCAL_GENERATED_SOURCES := $(generated-sources-dir)/pfw_wrap.cxx $(generated-sou
 
 LOCAL_EXPORT_C_INCLUDE_DIRS := $(generated-sources-dir)
 
+# Get the interpreter ld options.
+ifeq ($(HOST_OS), darwin)
+    # Contrary to linux, on darwin, a python 64 bit executable is installed
+    # in the x86 prebuild directory,
+    # As all host libraries are 32 bit in android. We can not link and host
+    # python module against the prebuild python library.
+    #
+    # As a *dirty* workaround, use the system's python configuration and hope
+    # it will be compatible with the prebuild python interpreter used at runtime.
+    # To summarize the prebuild python (64 bit?) interpreter will load a
+    # python native module (32bit) linked with the host (32 bit ?) python library.
+    LOCAL_LDLIBS += $(shell python-config --ldflags)
```

Comment:
The only differences between LDFLAGS and LDLIBS is there place in ld command line. As you can see in build/core/definitions.mk L1370 and L1380 LDFLAGS is added athe be begining and LDLIBS is added at the end.
As python-config flags must have precedence, add them at the end.